deployment/helm/mosquitto/values.yaml

```diff
@@ -0,0 +1,108 @@
+nameOverride: "mosquitto"
+fullnameOverride: ""
+
+# mosquitto image repo, tag and pull policy
+image: 
+  repository: eclipse-mosquitto
+  tag: 2.0.15
+  pullPolicy: IfNotPresent
+
+# mosquitto default MQTT port
+defaultPort: 1883
+tlsPort: 8883
+
+# deployment replica count
+replicaCount: 1
+
+# mosquitto pod resources collection
+resources:
+# We usually recommend not to specify default resources and to leave this as a conscious
+# choice for the user. This also increases chances charts run on environments with little
+# resources, such as Minikube. If you do want to specify resources, uncomment the following
+# lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  requests:
+    cpu: "50m"
+  limits:
+    memory: "128Mi"
+    cpu: "500m"
+
+# mosquitto deployment nodeSelectors{
+nodeSelector: {}
+
+# mosquitto pod tolerations
+tolerations: []
+
+# mosquitto pod affinity
+affinity: {}
+
+# mosquitto service type
+service:
+  type: LoadBalancer
```

Comment:
should this be configurable? Imagine, someone changes the type, then the whole deployment of mosquito gets a different sense, doesn´t it?

Reply:
Renamed 'service' to 'tlsService' to make it clearer. Folks could still decide to use ClusterIP if they don't want to expose a LoadBalancer outside the cluster, but with the scripts setup it will always default to LB as we require it for bridging.